platform/linux-generic/test/pktio_ipc/pktio_ipc_run.sh

```diff
@@ -17,9 +17,9 @@ PATH=$(dirname $0):$PATH
 PATH=$(dirname $0)/../../../../platform/linux-generic/test/pktio_ipc:$PATH
 PATH=.:$PATH
 
-RUNTIME1=10
-RUNTIME2=5
-TIMEOUT=13
+RUNTIME1=3
+RUNTIME2=1
+TIMEOUT=3
```

Comment:
I wonder if this timeout is too short for some software emulator, as usual.

Reply:
Maybe, who knows. We will see and can fix it then. The test script should actually be improved so that it does not take longer than necessary when things complete fast but would have more generous timeout for when things are slow (e.g. it should not sleep for a fixed time but poll for the process exit periodically). But I did not bother to do that but just this simple change that seems to be ok in the CI and my own environments.